saleor/graphql/order/mutations/fulfillments.py

```diff
@@ -622,22 +646,137 @@ class Meta:
         error_type_class = OrderError
         error_type_field = "order_errors"
 
+    @classmethod
+    def _check_shipping_costs(cls, payments_to_refund):
```

Comment:
This wouldn't be needed at all if we decided to keep `include_shipping_costs` where it is.